templates/constraints/yml/snippets/azureAppServiceZipDeploy.mustache

```diff
@@ -0,0 +1,16 @@
+  # Deploy your application to Azure App Service using the zip deploy feature.
+  # For additional details, refer to https://aka.ms/zip-deploy-to-app-services.
+  - uses: azureAppService/zipDeploy
+    with:
+    {{#workingDirectory}}
+      workingDirectory: {{workingDirectory}}
+    {{/workingDirectory}}
+      # Deploy base folder
+      artifactFolder: .
+      # Ignore file location, leave blank will ignore nothing
+      ignoreFile: .webappignore
```

Comment:
Do we use webappignore or appserviceignore?

Reply:
I think the correct one is appserviceignore. I will confirm it and update it in another PR.